### ISSUE TITLE
Can I use this mod for generating the cities in unexplored area of my existing world??

### ISSUE THREAD

**Issue by LeonMeraki** [McJty/LostCities#306]
I played a world so long with my friends and want to take a little adventure, but we don't want to abandoned our world and to create a new one. So if the mod could allow the existing world generating the cities, how?? also I'm playing with my friends through forge server 1.12.2 
please and thank you!

**Comment by RobinRMC** [comment]
Yes, and I found out how. Contact me on Discord and I will explain it to you.

https://discordapp.com/invite/7QDQRrrZ44 (It's a server so I don't have to share my Discord name on here.)
